src/commands/create-apikey.ts

```diff
@@ -0,0 +1,62 @@
+import {BaseCommand} from '../lib'
+import {getEnvironment} from '../lib/environments'
+import {flags} from '@oclif/command'
+import fetch from 'node-fetch'
+import {cli} from 'cli-ux'
+
+export default class CreateApikey extends BaseCommand {
+  static description = 'Create an API key for a Backend by id'
```

Comment:
could you please add `static hidden = true`

This way it stays out of the documentation, and we can release it when we also add list and delete api keys